src/control/drpc/modules.go

```diff
@@ -84,10 +84,12 @@ const (
 	MethodPoolUpdateACL = C.DRPC_METHOD_MGMT_POOL_UPDATE_ACL
 	// MethodPoolDeleteACL is a ModuleMgmt method
 	MethodPoolDeleteACL = C.DRPC_METHOD_MGMT_POOL_DELETE_ACL
+	// MethodListCont is a ModuleMgmt method
 	// MethodDevStateQuery is a ModuleMgmt method
 	MethodDevStateQuery = C.DRPC_METHOD_MGMT_DEV_STATE_QUERY
 	// MethodSetFaultyState is a ModuleMgmt method
 	MethodSetFaultyState = C.DRPC_METHOD_MGMT_DEV_SET_FAULTY
+	MethodListContainers = C.DRPC_METHOD_MGMT_LIST_CONTAINERS
```

Comment:
Looks like something went wrong with the merge... I can fix this in a follow-on PR I have that adds another method though.

Reply:
I see - the comment placement for the list containers method got messed up. Sorry I missed this when doing the manual conflict resolution.